core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -198,6 +197,8 @@ public long getTimeOut() {
 
   public TabletServerBatchWriter(ClientContext context, BatchWriterConfig config) {
     this.context = context;
+    this.executor = ThreadPools.getGeneralScheduledExecutorService(this.context.getConfiguration());
+    this.failedMutations = new FailedMutations();
```

Comment:
It's been almost a month since I looked at this. I don't remember the reason why I moved it and it should work if this line was reverted. I'm not sure if there is reason to do it one vs the other. I can easily change it back if necessary.

Reply:
No worries--I wasn't suggesting a change, but rather just trying to figure out if I missed something special about that one variable.